facade/nodeFacade.go

```diff
@@ -428,6 +445,10 @@ func (nf *nodeFacade) GetPeerInfo(pid string) ([]core.QueryP2PPeerInfo, error) {
 
 // GetThrottlerForEndpoint returns the throttler for a given endpoint if found
 func (nf *nodeFacade) GetThrottlerForEndpoint(endpoint string) (core.Throttler, bool) {
+	if !nf.wsAntifloodConfig.WebServerAntifloodEnabled {
```

Comment:
[comment]

I think we could've avoided this if and the one above by using some sort of polymorphic func.
However, i think the overhead is too big for too small reward

Reply:
I would leave it as it is